distribution/resources/bin/micro-gw

```diff
@@ -248,32 +248,39 @@ if [ "$IS_BUILD_COMMAND" = true ] && [ "$CMD_PRO_NAME_VAL" != "" ] && [ "$MICRO_
     -Dcli.home=$MICROGW_HOME \
     org.wso2.apimgt.gateway.cli.cmd.Main "$@"
 
-    MICRO_GW_LABEL_PROJECT_DIR="$MICRO_GW_PROJECT_DIR/$CMD_PRO_NAME_VAL"
-    pushd $MICRO_GW_LABEL_PROJECT_DIR/gen > /dev/null
-        # clean the content of target folder
-        if [ -d "$MICRO_GW_LABEL_PROJECT_DIR/target" ]; then
-            rm -rf $MICRO_GW_LABEL_PROJECT_DIR/target
-        fi
-        # build the ballerina source code for the label
-        ballerina build src/ -o $CMD_PRO_NAME_VAL.balx --offline --experimental --siddhiruntime
-    popd > /dev/null
+
+
+
+
+    if [ $? -eq 0 ]
```

Comment:
@VirajSalaka Change for `.bat` file not here. You can check for ERRORLEVEL variable value `.bat` logic

Reply:
added.